AdapterInterface.php

```diff
@@ -75,7 +75,7 @@ public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentO
      *
      * @return null|Symfony\Cmf\Component\Routing\RouteObjectInterface
      */
-    public function findRouteForUri($uri);
+    public function findRouteForUri($uri, UriContext $uriContext);
```

Comment:
Note that the `$uri` is not necessarily the same as the URI in `UriContext#getUri`.